recipes/flac/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class FlacConan(ConanFile):
+    name = "flac"
+    description = "Free Lossless Audio Codec"
+    topics = ("conan", "flac", "codec", "audio", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xiph/flac"
+    license = ("BSD", "GPL", "LPGL")
+    requires = "ogg/1.3.4"
+    exports_sources = "CMakeLists.txt",
+    generators = "cmake",
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    _cmake = None
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def build_requirements(self):
+        self.build_requires("nasm/2.14")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "{}-{}".format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, 'src', 'libFLAC', 'CMakeLists.txt'),
+            'target_link_libraries(FLAC PRIVATE $<$<BOOL:${HAVE_LROUND}>:m>)',
+            'target_link_libraries(FLAC PUBLIC $<$<BOOL:${HAVE_LROUND}>:m>)')
```

Comment:
Why do you need PUBLIC if you won't export that cmake file?

Reply:
this package produces several libraries, which link to  the FLAC library. Without this change, they fail to link. The fix was merged upstream today: https://github.com/xiph/flac/pull/195

Reply:
Undertood. Thanks!